juan-aristizabal-bedoya/tarea11-routing/routing/src/pages/Domingo.jsx

```diff
@@ -0,0 +1,14 @@
+import React from "react";
+import Title from "../components/Title";
+import Information from "../components/Information";
+
+const Domingo = () => {
```

Comment:
@Juan-Aristizabal this is a very bad approach you should only have one "task" component and pass data using props